src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <section class="search__box">
+      <form
+        class="search-bar search-bar__big"
+        data-qa="big"
+      >
+        <input
+          class="search-bar__search search-bar__search--large"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        class="search-bar search-bar__small"
+        data-qa="small"
+      >
+        <input
+          class="search-bar__search search-bar__search--small"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
+    </section>
```

Comment:
It's great that you are using `section` tag for semantic structuring of your content. However, in this case, the `section` tag is not the best choice. The `section` tag represents a standalone section of the document, which has its own heading. In your case, it would be more appropriate to use the `div` tag to group your forms.